remote_attestation/rust/src/message.rs

```diff
@@ -14,6 +14,12 @@
 // limitations under the License.
 //
 
+// Messages used to implement the Enclave Key Exchange Protocol (EKEP).
```

Comment:
I think we should refer to it as the Remote Attestation Protocol
And the note that it's a EKEP inspired is in the docs

Reply:
I think it's helpful to delineate between the full Remote Attestation Protocol, and the key exchange. The full protocol includes attestation report, which is defined as a proto (and presumably not intended to be binary compatible).

Reply:
This protocol is not EKEP, but a simplified version, so using "EKEP" is not accurate. "Remote attestation handshake protocol" might be more accurate. Also, perhaps add a link to the readme where the protocol is described.

Reply:
Another point I forgot to mention: this protocol includes the remote attestation report, but does not specify the format (it is just an opaque binary blob from the protocol perspective). The format for the attestation report is platform- and implementation-dependent. We currently use a proto as a placeholder, but once once we have a real implementation using the appropriate real hardware it will likely be a different format.

Reply:
thx clarified it's simplified EKEP, not EKEP.